cleverhans/attacks_tf.py

```diff
@@ -16,31 +16,55 @@
 FLAGS = flags.FLAGS
 
 
-def fgsm(x, predictions, eps, clip_min=None, clip_max=None):
+def fgsm(x, predictions, eps=0.3, clip_min=None, clip_max=None):
+    warnings.warn("attacks_tf.fgsm is deprecated and will be removed on "
```

Comment:
since "FGSM" is an acronym used in the literature, I'd rather not deprecate it, and just keep the wrapper around.